packages/devtools_app/lib/src/logging/flutter/logging_screen.dart

```diff
@@ -68,13 +72,23 @@ class _LoggingScreenState extends State<LoggingScreenBody> {
         ],
       ),
       Expanded(
-        child: LogsTable(
-          data: controller.data,
+        child: Split(
```

Comment:
do we not think expandable table children are practical? showing a separate details view in the existing logging page was more for expediency than by design. I'd hope we can do better in Flutter and actually show the details inline.

Reply:
It is doable, just out of scope for this PR, as discussed offline.